src/modules/search/components/SearchBar/SearchBar.test.js

```diff
@@ -1,158 +1,273 @@
+      wrapper.update();
+      jest.runAllTimers();
+    })
+    expect(wrapper).toContainMatchingElement('.accounts')
   });
 
-  it('should redirect to a different page if user do a click on selected row for address', () => {
-    wrapper.find('.search-input input').at(0).simulate('change', { target: { value: '123456L' } });
-    jest.advanceTimersByTime(500);
-    wrapper.update();
-    expect(props.suggestions.loadData).toBeCalled();
-
-    const newProps = {
-      ...props,
-      suggestions: {
-        ...props.suggestions,
-        data: {
-          ...props.suggestions.data,
-          addresses: [
-            {
-              summary: {
-                address: '123456L',
-                title: 'John',
-                balance: '120',
-              },
-            },
-          ],
+  it('should redirect to a different page if user do a click on selected row for transaction', () => {
+    useSearch.mockReturnValue({
+      addresses: [],
+      delegates: [],
+      transactions: [
+        {
+          params: {
+            data: 'testing',
+          },
+          id: '123456123234234',
+          moduleCommand: 'token:transfer',
         },
-      },
-    };
-    wrapper = mountWithRouter(SearchBar, newProps);
-    wrapper.find('.account-row').at(0).simulate('click');
+      ],
+      blocks: [],
+      isLoading: false
+    })
+    const wrapper = mountWithQueryClient(SearchBar, props)
+
+    act(() => {
+      wrapper.find('.search-input input').at(0).simulate('change', { target: { value: '123456123234234' } });
+    })
+
+    wrapper.find('.search-transaction-row').at(0).simulate('click');
     expect(props.history.push).toBeCalled();
-    expect(props.suggestions.clearData).toBeCalled();
   });
 
-  it('should redirect to a different page if user do a click on selected row for transaction', () => {
-    wrapper.find('.search-input input').at(0).simulate('change', { target: { value: '123456123234234' } });
-    jest.advanceTimersByTime(500);
-    wrapper.update();
-    expect(props.suggestions.loadData).toBeCalled();
-
-    const newProps = {
-      ...props,
-      suggestions: {
-        ...props.suggestions,
-        data: {
-          ...props.suggestions.data,
-          transactions: [
-            {
-              params: {
-                data: 'testing',
-              },
-              id: '123456123234234',
-              moduleCommand: 'token:transfer',
-            },
-          ],
+  it('should uses keyboard navigation to select search result for delegates', () => {
+    useSearch.mockReturnValue({
+      addresses: [],
+      delegates: [
+        {
+          address: '123456L',
+          username: 'genesis_10',
+          rank: 34,
+          rewards: 23423,
+          vote: 123,
         },
-      },
-    };
-    wrapper = mountWithRouter(SearchBar, newProps);
-    wrapper.find('.search-transaction-row').at(0).simulate('click');
+        {
+          address: '123457L',
```

Comment:
I think this address value `123457L` can be placed in a variable in the global scope so that the variable can be re-used. I observed its used in a lot of places within this test suite